pump/storage/storage_test.go

```diff
@@ -257,17 +248,21 @@ func (as *AppendSuit) TestReadWriteGCTS(c *check.C) {
 
 // test helper to write binlogNum binlog to append
 func populateBinlog(b Log, append *Append, prewriteValueSize int, binlogNum int32) {
```

Comment:
add comment on how  's behavior  when `prewriteValueSize`  <= 0,  and seems no where use it as `prewriteValueSize > 0 ` now for `fixed stype` now?